src/healthkitui.cs

```diff
@@ -29,4 +29,17 @@ interface HKActivityRingView {
 		[Export ("setActivitySummary:animated:")]
 		void SetActivitySummary ([NullAllowed] HKActivitySummary activitySummary, bool animated);
 	}
+
+	[iOS (17, 0), MacCatalyst (17, 0)]
+	[Category]
+	[BaseType (typeof (HKHealthStore))]
+	interface HKHealthStore_UIViewController {
+
+		[Export ("authorizationViewControllerPresenter")]
+		[return: NullAllowed]
+		UIViewController GetAuthorizationViewControllerPresenter ();
+
+		[Export ("setAuthorizationViewControllerPresenter:")]
+		void SetAuthorizationViewControllerPresenter ([NullAllowed] UIViewController authorizationViewControllerPresenter);
+	}
```

Comment:
I vote for doing this inline in the 'HKHealthStore' class rather than a category in this file. I'll let @rolfbjarne or @dalexsoto  decide.

Reply:
Yah I think I agree, inline this as a property inside `interface HKHealthStore` 

```csharp
[NullAllowed, Export ("authorizationViewControllerPresenter")]
UIViewController AuthorizationViewControllerPresenter { get; set; }
```

Reply:
Yes, that would actually allow us to make it a property instead of a Set/Get method pair.